Formula/nimbus-eth2.rb

```diff
@@ -0,0 +1,49 @@
+class NimbusEth2 < Formula
+  desc "Nim implementation of the Ethereum 2.0 blockchain"
+  homepage "https://nimbus.guide/"
+  # pull from git tag to get submodules
+  url "https://github.com/status-im/nimbus-eth2.git",
+      tag:      "v1.5.5",
+      revision: "f0f9735955070d297b33fbec466537cb7029e0cc"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make"
```

Comment:
This will create a debug build by default, not a release build.

Reply:
Nimbus is build in release mode by default.